### DISCUSSION TITLE
Conditionally switching workspaces

### DISCUSSION THREAD

**Discussion by ashutoshbsathe** [hyprwm/Hyprland#7429] [Q&A] [answered]
Hi,

I want to switch to either the numbered workspace or the previous one depending on my current workspace.

e.g. if my current workspace is 1 and I tap "mod + 1", then I should switch to the "previous" workspace but if my current workspace is NOT 1, then tapping "mod + 1" should take me to workspace 1

How do I achieve this in my config? As far as I can tell, there is no dispatcher that lets me do this. I know that there are `previous` and `previous_per_monitor` parameters in https://wiki.hyprland.org/Configuring/Dispatchers/#workspaces but I am not sure how I can tie the "if else" part in my binds.

**Comment by dryan-vallik** [comment, marked as answer]
https://wiki.hyprland.org/Configuring/Variables/#Binds

binds {
    workspace_back_and_forth = on
}

**Reply by ashutoshbsathe** [reply]
That was embarrassingly simple.  Thanks for the quick response!
